pkgs/tools/filesystems/bcachefs-tools/default.nix

```diff
@@ -65,20 +57,27 @@ stdenv.mkDerivation (finalAttrs: {
     udev
   ] ++ lib.optional fuseSupport fuse3;
 
-  # FIXME: Try enabling this once the default linux kernel is at least 6.7
-  doCheck = false; # needs bcachefs module loaded on builder
-  checkFlags = [ "BCACHEFS_TEST_USE_VALGRIND=no" ];
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    src = finalAttrs.src;
+    hash = "sha256-RsRz/nb8L+pL1U4l6RnvqeDFddPvcBFH4wdV7G60pxA=";
+  };
 
   makeFlags = [
-    "PREFIX=${placeholder "out"}"
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
```

Comment:
Are you sure this is right? https://github.com/NixOS/nixpkgs/issues/65718

Reply:
So a solution for this would be adding autotools to inputs and then using `installFlags = [ "localstatedir=${placeholder "out"}/var" ]; ` ?

I have not encountered any issues during normal usage with the new binary though.

Reply:
No, because this package doesn't use autotools, and localstatedir would only cover that one directory, not the whole thing.

The right thing to do is generally set PREFIX, and not set DESTDIR, like it was before.  Why did you change it?

Reply:
Looks like it was a oversight on my part. I was trying to cleanup this config by looking at the build.nix from the repository. It modify any existing behaviour at first glance, so I thought it was okay to include the change.

I'll revert this change.